spring-integration-http/src/main/java/org/springframework/integration/http/dsl/AsyncHttpMessageHandlerSpec.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.http.dsl;
+
+import org.springframework.expression.Expression;
+import org.springframework.expression.common.LiteralExpression;
+import org.springframework.http.client.AsyncClientHttpRequestFactory;
+import org.springframework.integration.expression.ValueExpression;
+import org.springframework.integration.http.outbound.AsyncHttpRequestExecutingMessageHandler;
+import org.springframework.util.Assert;
+import org.springframework.web.client.AsyncRestTemplate;
+
+import java.net.URI;
+
+/**
+ * The {@link org.springframework.integration.dsl.MessageHandlerSpec} implementation for the {@link org.springframework.integration.http.outbound.AsyncHttpRequestExecutingMessageHandler}.
```

Comment:
We don't need fully qualified class name for the `AsyncHttpRequestExecutingMessageHandler`.
It is imported for the class logic any way.